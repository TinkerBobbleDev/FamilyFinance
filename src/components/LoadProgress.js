import React from 'react';
import {withStyles} from 'material-ui/styles';

import {CircularProgress} from 'material-ui/Progress';

const styles = theme => ({
    progress: {
        margin: `0 ${theme.spacing.unit * 2}px`
        
    },
    divclass:{
        textAlign:'center',
    }
});

const LoadProgress = (props) => {
    const {classes} = props;
    return (
        <div className={classes.divclass}>
            <CircularProgress className={classes.progress} size={100}/>
        </div>
    );
}

export default withStyles(styles)(LoadProgress);