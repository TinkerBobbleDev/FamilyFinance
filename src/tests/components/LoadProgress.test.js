import React from "react";
// import ReactShallowRenderer from "react-test-renderer/shallow";
import LoadProgress from "../../components/LoadProgress";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

test("should render LoadProgress correctly", () => {
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<LoadProgress />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
  const wrapper = shallow(<LoadProgress />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
