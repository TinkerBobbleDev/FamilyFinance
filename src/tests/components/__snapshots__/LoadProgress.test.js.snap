// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadProgress correctly 1`] = `
<LoadProgress
  classes={
    Object {
      "divclass": "LoadProgress-divclass-2",
      "progress": "LoadProgress-progress-1",
    }
  }
/>
`;
